<launch>
  <node pkg="trans_pointcloud" type="trans_pointcloud" name="trans_pointcloud">

    <!-- resolution in meters per pixel -->
    <param name="octo_resolution" value="0.5" />

    <!-- name of the fixed frame, needs to be "/map" for SLAM -->
    <param name="frame_id" type="string" value="/rslidar" />

    <!-- max range / depth resolution of the kinect in meter -->
    <param name="bound_xy" value="20" />

    <!-- max/min height for occupancy map, should be in meters -->
    <param name="bound_highz" value="30" />
    <param name="bound_lowz" value="-30" />

    <param name="sub_topic" type="string" value="/points_raw" />
    <param name="pub_topic" type="string" value="/octomap_full" />
 
  </node>
  
  <!-- RViz -->
  <node pkg="rviz" type="rviz" name="$(anon rviz)" respawn="false" output="screen" args="-d $(find trans_pointcloud)/rviz/OctomapShow.rviz"/>
</launch>
